ni/test/test_bpo.py

```diff
@@ -61,7 +61,7 @@ def get(self, url):
         return self
 
     async def __aenter__(self):
-        self.session = aiohttp.ClientSession(loop=self.loop)
+        self.session = aiohttp.ClientSession(loop=self.loop, trust_env=True)
```

Comment:
Note: I had to add this option in order to be able to use env variables for setting a proxy, see https://docs.aiohttp.org/en/stable/client_advanced.html?highlight=proxy#proxy-support.

I can remove it if you think we shouldn't trust env variables (even if we're just testing).

Reply:
Yeah, please remove it just to be safe.